.changeset/beige-jobs-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+"eslint-plugin-svelte": minor
+---
+
+feat(no-unused-class-name): added a whitelist option
```

Comment:
I'm not familiar with it, but I think it's best to avoid using the term `whitelist`.
I prefer to simply use the term allowedClassNames option.